airbyte-api/src/main/openapi/config.yaml

```diff
@@ -848,6 +848,8 @@ components:
           $ref: "#/components/schemas/SourceSpecificationId"
         sourceId:
           $ref: "#/components/schemas/SourceId"
+        documentation:
```

Comment:
```suggestion
        documentationUrl:
```

Reply:
Also, can we specify that the format of this string will be a URI? it's [available in JSONSchema](https://json-schema.org/understanding-json-schema/reference/string.html#uri-template) but I'm not sure if it's available in OpenAPI